x/concentrated-liquidity/keeper.go

```diff
@@ -48,3 +50,17 @@ func (k Keeper) SetParams(ctx sdk.Context, params types.Params) {
 func (k *Keeper) SetPoolManagerKeeper(poolmanagerKeeper types.PoolManagerKeeper) {
 	k.poolmanagerKeeper = poolmanagerKeeper
 }
+
+// GetNextPositionId returns the next position id.
+func (k Keeper) GetNextPositionId(ctx sdk.Context) uint64 {
+	store := ctx.KVStore(k.storeKey)
+	nextPositionId := gogotypes.UInt64Value{}
+	osmoutils.MustGet(store, types.KeyNextGlobalPositionId, &nextPositionId)
+	return nextPositionId.Value
+}
+
+// SetNextPositionId sets next position Id.
+func (k Keeper) SetNextPositionId(ctx sdk.Context, positionId uint64) {
+	store := ctx.KVStore(k.storeKey)
+	osmoutils.MustSet(store, types.KeyNextGlobalPositionId, &gogotypes.UInt64Value{Value: positionId})
```

Comment:
Can you please integrate this into genesis as well?

Reply:
Integrated with tests

Reply:
https://github.com/osmosis-labs/osmosis/pull/4655/commits/e4ff8f8f92cfea1626b5239c11aabbf24f036556
https://github.com/osmosis-labs/osmosis/pull/4655/commits/09d779ca295574985b51cdb0f0ad08d13f1e19a0